src/pages/exchanges/Exchange.vue

```diff
@@ -29,4 +46,10 @@ export default Vue.extend({
     ...mapActions('exchanges', ['getExchange'])
   }
 });
-</script>
\ No newline at end of file
+</script>
+<style>
+.q-card {
+  width: 300px;
```

Comment:
https://quasar.dev/layout/grid/introduction-to-flexbox

Aquí explica como hacer un layout de elementos. Creo que deberíamos especificar las clases col-* tipo lo que ciden en https://quasar.dev/layout/grid/introduction-to-flexbox#Responsive-Design, y no poner un ancho fijo.